chapters/synchronous.tex

```diff
@@ -1083,6 +1092,13 @@ \subsection{Clocked Discrete-Time and Clocked Discretized Continuous-Time Partit
 Such a partition has to be solved with a \emph{solver method} of \cref{solver-methods}.
 When \lstinline!previous(x)! is used on a continuous-time state variable \lstinline!x!, then \lstinline!previous(x)! uses the start value of \lstinline!x! as value for the first clock tick.
 
+The use of the operator sample from \cref{event-related-operators-with-function-syntax} in a clocked partition is problematic.
+Diagnostics are appropriate especially if it is intended to generate sampling faster than the clocks, and otherwise the sampling should ideally be adjusted to the clock ticks.
```

Comment:
To me it seems that this is a normative change that should require a language group decision, as the previous state didn't make clear that the problematic `sample` really is allowed.

Reply:
I can separate out that to another PR, to have a clearer base for discussion.

Reply:
Sounds like a good idea.

Reply:
This has now been done.

Reply:
Which makes this PR unproblematic.

Reply:
Moved to #3099.  Resolving.